README.md

```diff
@@ -179,6 +181,19 @@ forge build
 forge test
 ```
 
+## Contributions
+Before you submit your PR, run all of the following and commit the changes:
+```bash
+# update gas snapshots
+yarn test:gas
+
+# make sure all tests pass
+yarn test
```

Comment:
technically its more efficient for them to just run 1 test command that updates gas snapshots - these 2 commands mean they run the gas tests twice is a waste of time

Reply:
we could add another command that does
`UPDATE_SNAPSHOT=1 yarn test` all in 1